src/xpcc/architecture/platform/devices/stm32/stm32f100-c_r_v-8_b.xml

```diff
@@ -66,7 +66,7 @@
     <driver type="i2c" name="stm32" instances="1,2"/>
     <driver type="spi" name="stm32" instances="1,2"/>
     <driver type="spi" name="stm32_uart" instances="1,2,3"/>
-    <driver type="timer" name="stm32" instances="1,2,3,4,6,7,15,16,17"/>
+    <driver type="timer" name="stm32" instances="1,2,3,4,6,7"/>
```

Comment:
You are right, those should exists. I removed them due to compiler errors. But those where not due to missing timers but due to missnamed interrupts in driver.